docs/SUMMARY.md

```diff
@@ -1,6 +1,22 @@
 * [Home](index.md)
+* [Install](install/)
```

Comment:
does not exist; consider markdown file with tabs for conda, pip per execution backend:

```
{% for bend in ["duckdb", "bigquery", "postgres", "mysql", "snowflake" %}
=== "pip"
    === "{{ bend }}"
        ~~~sh
        pip install ibis-framework[{{ bend }}]
        ~~~

{% for mgr in ["conda", "mamba"] %}
=== "{{ mgr }}"
    === "{{ bend }}"
        ~~~sh
        {{ mgr }} install -c conda-forge ibis-{{ bend }}
        ~~~

{% endfor %}
{% endfor %}
```